go.mod

```
@@ -1,18 +1,20 @@
-module prctl
+module preq
 
-go 1.12
+go 1.14
```

```suggestion
go 1.14
```
keep up with the times